src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -131,7 +133,8 @@ public void start(Map<String, String> props, JestClient client) {
           .setLingerMs(lingerMs)
           .setRetryBackoffMs(retryBackoffMs)
           .setMaxRetry(maxRetry)
-          .setDropInvalidMessage(dropInvalidMessage);
+          .setDropInvalidMessage(dropInvalidMessage)
```

Comment:
What happens to invalid messages ?
If we have an invalid message and do not want to just drop it, what happens then ?

Reply:
This pull request does not change any behavior on invalid message. Basically the connector provides a config on that - you can choose to ignore invalid message, or stop the Sink Task once invalid message occurs.